frontend/packages/dev-console/src/components/catalog/details/CatalogDetailsPanel.tsx

```diff
@@ -46,14 +47,16 @@ const CatalogDetailsPanel: React.FC<CatalogDetailsPanelProps> = ({ item }) => {
               )}
             </PropertiesSidePanel>
             <div className="co-catalog-page__overlay-description">
-              <SectionHeading text={t('devconsole~Description')} />
-              {!details?.descriptions?.length && description && <p>{description}</p>}
-              {details?.descriptions?.map((desc, index) => (
-                <React.Fragment key={index}>
-                  {desc.label && <SectionHeading text={desc.label} />}
-                  {desc.value}
-                </React.Fragment>
-              ))}
+              <Stack hasGutter>
```

Comment:
@christianvogt @serenamarie125 There is a spacing issue please take a look at before and after pics:

before:
<img width="904" alt="Screen Shot 2020-11-29 at 13 28 30" src="https://user-images.githubusercontent.com/24938324/100540600-ebcb4980-3246-11eb-8a11-cd5b23dd317e.png">

-------

after:
<img width="895" alt="Screen Shot 2020-11-29 at 13 28 53" src="https://user-images.githubusercontent.com/24938324/100540604-eff76700-3246-11eb-9b95-b3b114ccfda0.png">

Reply:
cc @rohitkrai03 
This change will affect all other providers. I believe they are including their own spacing. Perhaps with this change we can remove that spacing in the providers.

Reply:
Yeah, this does affect all the other providers. It would be better if we remove the spacing from the providers along with this change. The above change causes below spacing issue with current catalog providers - 

![Screenshot from 2020-11-30 21-54-35](https://user-images.githubusercontent.com/6041994/100635990-376d1880-3357-11eb-8c38-e39884f540bc.png)
![Screenshot from 2020-11-30 21-55-24](https://user-images.githubusercontent.com/6041994/100635998-3936dc00-3357-11eb-9ea5-869d823c8679.png)
![Screenshot from 2020-11-30 21-56-58](https://user-images.githubusercontent.com/6041994/100636000-39cf7280-3357-11eb-94aa-2d189dc6e2ce.png)

Reply:
@matthewcarleton

Reply:
+1 from me on the look of the spacing on the "after" I def agree we should ship one way of spacing. IF there are edge cases where <hr/> are used it would be nice to just pull those IMO :)